app/views/forms/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1 class="govuk-heading-l">Edit form</h1>
+
+<% if flash[:message] %>
+  <p class="govuk-body">
+    <%= flash[:message] %>
+  </p>
+<% end %>
+
+<%= form_with model: @form do |form| %>
+  <div class="govuk-form-group">
+    <h1 class="govuk-label-wrapper">
+      <%= form.label :name, "What is the name of your form?", class: "govuk-label" %>
+    </h1>
+    <div id="form-title-hint" class="govuk-hint">
+      The form name will be shown at the top of each page of the form. Use a name that describes what the form will help people to do. For example 'Apply for a juggling licence'.
+    </div>
+    <%= form.text_field :name, class: "govuk-input", 'aria-describedby': "form-title-hint" %>
+  </div>
+
+  <div class="govuk-form-group">
+    <label class="govuk-label" for="email">
+      Email address
+      <%= form.label :email, "What email should responses be sent to?", class: "govuk-label" %>
+    </label>
+    <div id="email-hint" class="govuk-hint">
+      This should be a departmental shared inbox.
+    </div>
+    <%= form.email_field :email, class: "govuk-input", 'aria-describedby': "email-hint", spellcheck: "false", autocomplete: "email" %>
```

Comment:
this should be called `submission_email` i think - looking at https://github.com/alphagov/forms-api/blob/main/lib/server.rb#L13-L17